src/faillog.c

```diff
@@ -163,6 +163,10 @@ static void print_one (/*@null@*/const struct passwd *pw, bool force)
 	}
 
 	tm = localtime (&fl.fail_time);
+	if (!tm) {
+		fprintf (stderr, "Cannot read time from faillog.\n");
+		exit (EXIT_FAILURE);
```

Comment:
Thank you for the patch.  However, in keeping with the rest of this function it would be better to return rather than exit.  Could you update the patch to do that?